docs/usage/exporting_models.md

```diff
@@ -84,6 +84,17 @@ as an optional alternative which implements ISO 8601 time diff encoding.
 See [below](#custom-json-deserialisation) for details on how to use other libraries for more performant JSON encoding
 and decoding.
 
+## `model.construct()`
```

Comment:
best to move this to https://pydantic-docs.helpmanual.io/usage/models/#helper-functions 

That section will need more work, but at least if put it there for now it will end up in the right place